provider/internal/device/ios/resource_test.go

```diff
@@ -25,8 +25,8 @@ resource "cdo_ios_device" "test" {
 	name = "{{.Name}}"
 	socket_address = "{{.SocketAddress}}"
 	connector_type = "{{.SdcType}}"
-	username = "{{.Username}}"
-	password = "{{.Password}}"
+	username = "{{.EncryptedUsername}}"
```

Comment:
The user shouldn't be providing the encrypted username and password here right?

Reply:
yes thats correct, hmm this should fail the tests, let me check...

Reply:
Fixed CI in https://github.com/CiscoDevNet/terraform-provider-cdo/pull/14